__tests__/app.test.js

```diff
@@ -55,4 +55,54 @@ describe('/api', () => {
         })
     })
 
-    
\ No newline at end of file
+describe('/api/articles/:article_id', () => {
+    test('GET 200: responds with an article object in the correct format', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body}) => {
+            const {article} = body
+            expect(typeof article).toBe('object')
+            expect(typeof article[0].article_id).toBe('number')
+            expect(typeof article[0].title).toBe('string')
+            expect(typeof article[0].topic).toBe('string')
+            expect(typeof article[0].author).toBe('string')
+            expect(typeof article[0].body).toBe('string')
+            expect(typeof article[0].created_at).toBe('string')
+            expect(typeof article[0].votes).toBe('number')
+            expect(typeof article[0].article_img_url).toBe('string')
+        })
+    })
+    test('GET 200: responds with an article object with the correct properties', () => {
```

Comment:
This second happy path test would be fine on it's own without the one above as this test also proves everything tested in the test above